app/src/main/java/edu/uc/coffeens/stocktracker/services/StockService.kt

```diff
@@ -15,12 +16,12 @@ class StockService {
      */
     fun fetchStocks(): MutableLiveData<ArrayList<Stock>> {
 
-        var _stocks = MutableLiveData<ArrayList<Stock>>()
+        val _stocks = MutableLiveData<ArrayList<Stock>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(IStockDAO::class.java)
         val call = service?.getAllStocks()
         call?.enqueue(object : Callback<ArrayList<Stock>> {
             override fun onFailure(call: Call<ArrayList<Stock>>, t: Throwable) {
-                print("Unable to retrieve service response.")
+                Log.d(this::class.java.simpleName, t.toString())
```

Comment:
instead of printing the error to the console, I went ahead and created a log that will output the throwable error into the log file of the project.

Reply:
Good idea.  I'd go one step further and make it Log.e (error) instead of Log.d (debug)